### PR TITLE
Added categorical data example

### PR DESCRIPTION
**Pull request by Joao-Dionisio**

### CODE REVIEW

**Review by mmghannam — APPROVED** [review]

File: examples/finished/categorical_data.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+from pyscipopt import Model
+
```
[the remark this answers is not in the document]
Comment (Joao-Dionisio) [reply examples/finished/categorical_data.py:16]: Done!

**Comment by Jiunixo** [comment]
Hello,
Thank you for your work and all these nice and pedagogic examples.
Trying this specific one, I think that the following constraints are missing :  
- each employee must be assigned to exactly one shift

```
# Each employee must be assigned to exactly one shift
for e in employees:
    model.addCons(sum(x[e, s] for s in shift_to_int.values()) == 1)
```
So, if you test the original example with cost = {
    "Alice":   [2,4,1],
    "Bob":     [3,3,2],
    "Charlie": [3,3,3]
},
then Bob is assigned to 2 shifts and Charlie to none.
I can create issue and PR if it helps.

**Comment by Joao-Dionisio** [comment]
Oops, you're exactly right @Jiunixo, thank you for pointing it out! If you wouldn't mind creating a PR, I can merge it straight away :)

**Comment by Jiunixo** [comment]
It's done there https://github.com/scipopt/PySCIPOpt/pull/939
I'm not very used to GitHub, don't hesitate to tell me if something goes wrong : )

**Pull request merged**